components/Order/basic_usage.rst

```diff
@@ -0,0 +1,251 @@
+    $shippingFees->setAmount(1000);
+    $shippingFees->lock();
+
+    $discount = new Adjustment();
+    $discount->setAmount(-500);
+
+    $order->addItem($tshirt);
+    $order->addAdjustment($shippingFees);
+    $order->addAdjustment($discount);
+    $order->removeAdjustment($shippingFees);
+    $order->calculateTotal();
+    $order->getTotal();
+
+    // Output will be 5499.
+
+Order
+-----
+
+Each order has 2 main identifiers, an ID and a human-friendly number. You can access those by calling ``->getId()`` and ``->getNumber()`` respectively.
+The number is mutable, so you can change it by calling ``->setNumber('E001')`` on the order instance.
+
+Order Totals
+~~~~~~~~~~~~
+
+.. note::
+
+    All money amounts in Sylius are represented as "cents" - integers.
+
+An order has 3 basic totals, which are all persisted together with the order.
+
+The first total is the *items total*, it is calculated as the sum of all item totals.
+
+The second total is the *adjustments total*, you can read more about this in next chapter.
+
+.. code-block:: php
+
+    <?php
+
+    echo $order->getItemsTotal(); // 1900.
+    echo $order->getAdjustmentsTotal(); // -250.
+
+    $order->calculateTotal();
+    echo $order->getTotal(); // 1650.
+
+The main order total is a sum of the previously mentioned values.
+You can access the order total value using the ``->getTotal()`` method.
+
+Recalculation of totals can happen by calling ``->calculateTotal()`` method, using the simplest math. It will also update the item totals.
+
+Items Management
+~~~~~~~~~~~~~~~~
+
+The collection of items (Implementing the ``Doctrine\Common\Collections\Collection`` interface) can be obtained using the ``->getItems()``.
+To add or remove items, you can simply use the ``addItem`` and ``removeItem`` methods.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Order\Model\Order;
+    use Sylius\Component\Order\Model\OrderItem;
+
+    $order = new Order();
+
+    $item1 = new OrderItem();
+    $item1->setName('Super cool product');
+    $item1->setUnitPrice(1999); // 19.99!
+    $item1->setQuantity(2);
+
+    $item2 = new OrderItem();
+    $item2->setName('Interesting t-shirt');
+    $item2->setUnitPrice(2549); // 25.49!
+
+    $order->addItem($item1);
+    $order->addItem($item2);
+    $order->removeItem($item1);
+
+Order adjustment
+~~~~~~~~~~~~~~~~
+
+The order holds adjustments.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Order\Model\Order;
+    use Sylius\Component\Order\Model\OrderItem;
+    use Sylius\Component\Order\Model\Adjustment;
+
+    $order = new Order();
+    $item = new OrderItem();
+    $adjustment = new Adjustment();
+
+    $adjustment->setAmount(1200);
+    $adjustment->setLabel('tax');
+
+    $item->setOrder($order);
+    $item->setQuantity(2);
+    $item->setUnitPrice(2000);
+
+    $order->addAdjustment($adjustment);
+    $order->addItem($item);
+    $order->calculateTotal();
+
+    $order->getTotal(); //5200
```

Comment:
I think we can skip this part, because it is shown in all other examples with adjustments etc. A bit of duplication imo. Agree?